<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>question</class>
 <widget class="QWidget" name="question">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>437</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="AcrylicFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>397</width>
       <height>441</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>看中文写英文</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="close_btn">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="chn_label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>中文</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="eng_frame_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>300</width>
        <height>80</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="eng_edit">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>281</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>input</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="AcrylicFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>397</width>
       <height>451</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="close_btn_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="eng_label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>英文</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="B_btn">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>360</y>
        <width>300</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>C. 正确的</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>看英文选中文</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="C_btn">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>300</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>B. 错误的</string>
      </property>
     </widget>
     <widget class="QPushButton" name="A_btn">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>300</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>A. 错误的</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AcrylicFrame</class>
   <extends>QFrame</extends>
   <header location="global">acrylicframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
